src/services/notification.service.js

```diff
@@ -74,4 +81,156 @@ export default class NotificationService {
       where: user
     });
   }
+
+  /**
+   * @param {object} request
+   * @return {function} create notifications
+   */
+  static async findManagerToNotify(request) {
+    const { lineManagerId } = await UserService.findUserByProperty({
+      id: request.userId
+    });
+
+    const { isInAppNotification, isEmailNotification } = await PreferenceService
+      .findPreference({ userId: lineManagerId });
+    const manager = await UserService.findUserByProperty({ id: lineManagerId });
```

Comment:
@erickyvand this line and the 90th line are confusing? what's the difference between the two?